python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -52,6 +52,15 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetEventTagsTable = db.Table(
+    "asset_event_tags",
+    SqlEventLogStorageMetadata,
+    db.Column("id", db.Integer, primary_key=True, autoincrement=True),
+    db.Column("event_id", db.Integer, db.ForeignKey("event_logs.id", ondelete="CASCADE")),
+    db.Column("key", db.Text),
+    db.Column("value", db.Text),
```

Comment:
should we have asset_key in here?  Might make it easier to do stat queries without doing joins?

Reply:
Makes sense. We'll be able to find out how many multi-dimensional partitions are filled by just querying this table, since we're only allowing tags on asset materializations at the moment